code/ship/ship.cpp

```diff
@@ -13281,29 +13281,27 @@ float ship_get_subsystem_strength( ship *shipp, int type )
  */
 void ship_set_subsystem_strength( ship *shipp, int type, float strength )
 {
-	float total_current_hits, diff;
 	ship_subsys *ssp;
 
 	Assert ( (type >= 0) && (type < SUBSYSTEM_MAX) );
-	if ( shipp->subsys_info[type].aggregate_max_hits <= 0.0f )
```

Comment:
Why was this condition dropped? I din't know much about the subsystem code so I'll need a little help while reviewing this.

Reply:
This condition prevents adjusting the subsystem strength if it is already at 0.  Removing it allows the function to be used to repair subsystems if desired.

Reply:
Ok, I guess that makes sense. Thanks for the clarification.